sdk/identity/identity/src/msal/nodeFlows/msalOpenBrowser.ts

```
@@ -166,13 +161,24 @@ export class MsalOpenBrowser extends MsalNode {
         }
       }
 
-      const abortSignal = options?.abortSignal;
-      if (abortSignal) {
-        abortSignal.addEventListener("abort", () => {
+      app.on("connection", (socket) => socketToDestroy.push(socket));
+
+      app.on("listening", () => {
+        const openPromise = this.openAuthCodeUrl(scopes);
+
+        const abortSignal = options?.abortSignal;
+        if (abortSignal) {
+          abortSignal.addEventListener("abort", () => {
+            cleanup();
+            reject(new Error("Aborted"));
```

Unrelated to the PR.. do you want to throw a better error?